BUILDING.md

```diff
@@ -603,3 +603,11 @@ platforms to avoid issues with Boost config files (introduced in Boost version
 to use Boost specified config files for their USD build, specify 
 -DBoost_NO_BOOST_CMAKE=OFF when running cmake.
 
+2. Windows and Python 3.8+
+After python 3.8 on windows, python will no longer search the whole path for
```

Comment:
Some tweaks, lmk what you think:

Python 3.8 and later on Windows will no longer search PATH for DLL dependencies.
Instead, clients can call `os.add_dll_directory(p)` to set paths to search.
By default on that platform USD will iterate over PATH and add all paths using
`os.add_dll_directory()` when importing Python modules. Users may override
this by setting the environment variable `USD_WINDOWS_DLL_PATH` to a PATH-like
string. If this is set, USD will use these paths instead.

Reply:
I took this wording exactly, but also added PXR_ to the env var name